getting-started/templates/pilot-sizing.yaml

```diff
@@ -703,6 +703,11 @@ webserver:
     targetCPUUtilizationPercentage: 80
     targetMemoryUtilizationPercentage: 90
 
+  # Increase default rate limit to adjust for reduced node count.
```

Comment:
Suggestion: 'Increases the default rate limit to adjust for a node count reduction.'